pennylane/math/single_dispatch.py

```diff
@@ -120,11 +120,17 @@ def _scatter_element_add_autograd(tensor, index, value):
     """In-place addition of a multidimensional value over various
     indices of a tensor. Since Autograd doesn't support indexing
     assignment, we have to be clever and use ravel_multi_index."""
+    pnp = _i("qml").numpy
     size = tensor.size
-    flat_index = _i("qml").numpy.ravel_multi_index(index, tensor.shape)
+    flat_index = pnp.ravel_multi_index(index, tensor.shape)
+    if pnp.isscalar(flat_index):
+        flat_index = [flat_index]
+    if pnp.isscalar(value) or len(pnp.shape(value)) == 0:
+        value = [value]

```

Comment:
can you remind me what the change here does?

Reply:
might be good to add to the changelog?

Reply:
Oh yes, will add it.

This allows us to use `scatter_element_add` with multiple indices and multiple values in one go :)